src/include/mallocMC/reservePoolPolicies/SimpleCudaMalloc_impl.hpp

```diff
@@ -27,6 +27,8 @@
 
 #pragma once
 
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
Can I implement this for CPU targets? E.g. by just using ::malloc()? For now I created a separate reservePolicy, but maybe we should just rename the whole policy into DeviceMalloc and use alpaka buffers. What do you think?

Reply:
Yes o CPU it can become a simple `::malloc` 

Maybe we should call this policy `SimpleMalloc` or `SystemMalloc` and use the cuda/hip way or ::malloc.

Reply:
I called it Malloc.